docs/conversion/sakai_20_mysql_conversion.sql

```diff
@@ -70,3 +70,15 @@ DROP TABLE PERMISSIONS_SRC_TEMP;
 ALTER TABLE SAM_ITEM_T ADD ISEXTRACREDIT TINYINT(1);
 ALTER TABLE SAM_PUBLISHEDITEM_T ADD ISEXTRACREDIT TINYINT(1);
 -- END SAK-34741
+
+-- SAK-42423 GOOGLEDRIVE
+CREATE TABLE GOOGLEDRIVE_USER (
+  sakaiUserId varchar(255) NOT NULL,
+  googleDriveName varchar(255) DEFAULT NULL,
+  refreshToken longtext,
+  googleDriveUserId varchar(255) DEFAULT NULL,
+  token longtext,
```

Comment:
How large can the token be?

Reply:
It's variable but I've seen some tokens taking up to 1200 chars

Reply:
Got it. Thanks